apps/api/src/app/widgets/usecases/get-subscriber-template-preference/get-subscriber-template-preference.usecase.ts

```diff
@@ -37,7 +37,10 @@ export class GetSubscriberTemplatePreference {
         template: responseTemplate,
         preference: {
           enabled: currSubscriberPreference.enabled,
-          channels: filterActiveChannels(currSubscriberPreference.channels, activeChannels),
+          channels: filterActiveChannels(
```

Comment:
Maybe worth adding tests for this in the existing tests too?

Reply:
Yes, almost ready. Will push soon.